plotter/heat.go

```diff
@@ -223,3 +224,60 @@ func (h *HeatMap) GlyphBoxes(plt *plot.Plot) []plot.GlyphBox {
 	}
 	return b
 }
+
+// RasterHeatMap implements the plotter interface. It is similar to HeatMap,
+// apart from that the plot method uses a raster-based method when drawing
+type RasterHeatMap struct {
+	HeatMap *HeatMap
+}
+
+// NewRasterHeatMap creates as new heat map plotter for the given data,
+// using the provided palette.
+func NewRasterHeatMap(g GridXYZ, p palette.Palette) *RasterHeatMap {
+	return &RasterHeatMap{
+		HeatMap: NewHeatMap(g, p),
+	}
+}
+
+// Plot implements the Plot method of the plot.Plotter interface.
+func (r *RasterHeatMap) Plot(c draw.Canvas, p *plot.Plot) {
+	cols, rows := r.HeatMap.GridXYZ.Dims()
+	img := image.NewRGBA64(image.Rectangle{
+		Min: image.Point{X: 0, Y: 0},
+		Max: image.Point{X: cols, Y: rows},
+	})
+
+	var pImg *Image
```

Comment:
I'd remove that line (and use a `:=` declaration+initialization below for the plot image)

Reply:
Yes, that's better